src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Presentation/RoslynCompletionSet.cs

```diff
@@ -12,16 +11,15 @@
 using Microsoft.VisualStudio.Language.Intellisense;
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Editor;
-using Roslyn.Utilities;
 using VSCompletion = Microsoft.VisualStudio.Language.Intellisense.Completion;
 using Microsoft.CodeAnalysis.Snippets;
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.Completion.Presentation
 {
-#if NEWCOMPLETION
-    internal sealed class CompletionSet3 : CompletionSet2
+#if DEV15
```

Comment:
Yeah, I thought about that too, I just got lazy :stuck_out_tongue_winking_eye:

Let's fix it up after.